auslib/web/admin/swagger/api.yaml

```diff
@@ -108,6 +131,110 @@ parameters:
     minimum: 0
     required: true
 
+  change_id_param:
+    name: change_id
+    in: path
+    description: Unique id to identify the identify the change
+    type: string
+    required: true
+
+  field_name_param:
+    name: field
+    in: path
+    description: name of DB column whose history is requested
+    type: string
+    required: true
+
+  userPermissionFormDataParam:
+    name: user_permission_request_body
+    in: body
+    description: Request body containing options and data version field values for manipulating user permission values in DB
+    required: true
+    schema:
+      allOf:
+        - $ref: '#/definitions/CSRFModel'
+        - type: object
+          properties:
+            data_version:
+              type: integer
+              format: int32
+              description: data_version of permission
+              minimum: 1
+              example: 1
```

Comment:
It is going to be checked explicitly in the method body.
**Reason**: considering the PUT operation's [method](https://github.com/aksareen/balrog/blob/user_then_releases/auslib/web/admin/views/permissions.py#L73),  it handles two cases: First whether the input permission [already exists](https://github.com/aksareen/balrog/blob/user_then_releases/auslib/web/admin/views/permissions.py#L77) (if yes then update it using data_version as required parameter). If it [does not](https://github.com/aksareen/balrog/blob/user_then_releases/auslib/web/admin/views/permissions.py#L92) then it create a new permission (data_version isn't required in the input).